### ISSUE TITLE
[Bug]: Shift Key causes input box to cancel and loose focus

### ISSUE THREAD

**Issue by placidic** [lowcoder-org/lowcoder#1101]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When editing an applicaion: When I interact with a input box, like when trying to rename a component, or typing in an initial value of a temporary state:

If i try to use a capital letter by holding the Shift Key (Right or Left), the shift key causes the input box to discard any changes and loose focus of the input.

This prevents any lables, or text, that uses Shift (Like special characters too)

### Expected Behavior

Shift allows using characters that are capital or special characters within text boxes

### Steps to reproduce

1. Create an App
2. Add a Temporary State
3. Try to set the value of the temporary state to :  "I'mData"  using the shift key to Capitalize
4. Try to rename a component with Capital Letters

### Environment

lowcoder cloud 2.4.5
lowcoder self hosted 2.4.5

**Comment by sonuame** [comment]
happening with me too. 
It happens only on "Temporary State"
Not able to rename the state name, or value. The moment you press "shift", it looses focus from the box

**Comment by sonuame** [comment]
I made this script to remove all key press events 
```javascript 
try { getEventListeners(document)['keypress'].forEach(e => { document.removeEventListener('keypress', e.listener, e.useCapture);}); } catch {}

try { getEventListeners(document)['keydown'].forEach(e => { document.removeEventListener('keydown', e.listener, e.useCapture);}); } catch {}

try { getEventListeners(document)['keyup'].forEach(e => { document.removeEventListener('keyup', e.listener, e.useCapture);}); } catch {}
```

Just run this multiple times (2-3 times) on developer window console.

**Comment by adnanqaops** [comment]
The above issue has been fixed in V2.4.7. It's not there anymore. So, closing out this issue.
